src/app.tsx

```diff
@@ -100,8 +102,18 @@ export const App = () => {
     setIsFailed(false);
     setIsSolved(false);
     setAttempt(0);
+    setDayWordCounter(dayWordCounter + 1);
+    setDayWordCounterDisplay(stringifyNumber(dayWordCounter + 1));
   };
 
+  const stringifyNumber = (n) => {
```

Comment:
@samedwards I would like is to move the `stringifyNumber` into its own utility class and call externally from `app.tsx` but I couldn't work out how to do this with ReactJS